library/private/animationManager.h

```diff
@@ -129,6 +149,10 @@ class animationManager
   double DeltaTime = 0;
   bool CurrentTimeSet = false;
 
+  // Dynamic options
+  bool Autoplay = false;
+  double SpeedFactor = 1.0;
```

Comment:
lets always uses these members in animationManager.cxx instead of using the options